CONTRIBUTING.rst

```diff
@@ -98,7 +98,7 @@ Adding new SPARQL queries to Scholia
      SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }  
    }
 
-4. Make a variable e.g. ``authorAwardsSparql`` that uses Jinja's templating system to load
+4. Make a variable e.g. ``authorAwardsSparql`` that uses `Jinja's <https://jinja.palletsprojects.com/>`_ templating system to load
```

Comment:
This is no longer necessary

Reply:
@egonw I don't have write access to your repo so maybe you can directly remove this

Reply:
@fnielsen, @cthoyt, oh, duh... sorry about that.

Both, I added a patch, see below, addressing the above point and tuning it to the updated approach.